docs/solutions/hypervisors/kvm/kvm-baremetal/variables.tf

```diff
@@ -0,0 +1,90 @@
+#Variables declaration file
+
+#OCI Environment variables
+variable "tenancy_ocid" {}
+variable "user_ocid" {}
+variable "compartment_ocid" {}
+variable "region" {}
+variable "fingerprint" {}
+variable "private_key_path" {}
+variable "ssh_user_private_key_path" {}
+variable "ssh_user_public_key_path" {}
+
+#Prefix to identify your resources
+variable "prefix" {
+  default = "samplekvm"
+}
+
+#availability_domain number For AD1 uses 1. For AD2, uses 2, For AD3, uses 3
+variable "availability_domain" {
+  default = "1"
+}
+
+#Cidr block for your VCN
+variable "vcn_cidr_block" {
+  default = "10.0.0.0/16"
+}
+
+#Cidr block for your subnet
+variable "kvm_host_subnet_cidr_block" {
+  default = "10.0.10.0/24"
+}
+
+#Oracle Linux 7.x Image OCID
+#Reference: https://docs.us-phoenix-1.oraclecloud.com/Content/Compute/References/images.htm
+variable "image_ocid" {
+  #Default Oracle Linux 7.4 image - Ashburn
+  default = "ocid1.image.oc1.iad.aaaaaaaaxrqeombwty6jyqgk3fraczdd63bv66xgfsqka4ktr7c57awr3p5a"
+}
+
+#KVM Host instance shape. Only BM Shapes are supported on this example
+variable "instance_bm_shape" {
+  default = "BM.Standard1.36"
+}
+
+#URL of your image file for downloading (you can place your image in the object storage!)
+variable "kvm_image_url" {
+  default = "https://<my-qcow2-image-url>"
```

Comment:
I would think that this shouldn't have a default, since it won't work unless you provide a different value.